.github/workflows/crowdin_action.yml

```
@@ -1,34 +1,31 @@
 name: Crowdin Action
 
-permissions:
-  contents: write
-  pull-requests: write
-
 on:
   push:
     branches:
       - main
   schedule:
-    - cron: "0 */12 * * *"
+    - cron: '0 */12 * * *'
 
 jobs:
   synchronize-with-crowdin:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.METAMASKBOT_CROWDIN_TOKEN }}
```

Nit: The comment may help explain why we're doing this, so nobody thinks this a mistake. We broke this once already in the extension as a result of this type of misunderstanding
```suggestion
          # Use PAT to ensure that the commit later can trigger status check workflows
          token: ${{ secrets.METAMASKBOT_CROWDIN_TOKEN }}
```

---

Thank you @Gudahtt !